roles/ceph-osd/tasks/pre_requisite.yml

```diff
@@ -5,12 +5,14 @@
     state: present
   when: ansible_os_family == 'Debian'
 
+- name: Install yum utils
```

Comment:
Please don't use capital letters.
Why do we want to install yum-utils?

This doesn't really look like a mandatory package for Ceph.
If there is anything else you'd like to see, it has to be part of your own infrastructure playbook.

👎